sdk/go/pulumi/generate/main.go

```diff
@@ -350,19 +349,11 @@ func main() {
 		f.Close()
 
 		gofmt := exec.Command("gofmt", "-s", "-w", fullname)
-		stderr, err := gofmt.StderrPipe()
+		out, err := gofmt.CombinedOutput()
```

Comment:
Given we we're just copying to os.Stdout I think this can actually just set `gofmt.Stdout = os.Stdout` and call Run.

Reply:
Done!